### PR TITLE
feat: implement Delete endpoint tests

### PR DESCRIPTION
**Pull request by radhus**
Implements basic Delete endpoint tests, identical to the Get endpoint tests.

### CODE REVIEW

File: internal/aiptest/deletion/exists.go [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+		if util.HasParent(scope.Resource) {
+			f.P("parent := ", ident.FixtureNextParent, "(t, false)")
+			f.P("created := fx.create(t, parent)")
+		} else {
+			f.P("created := fx.create(t)")
+		}
+		util.MethodDelete{
+			Resource: scope.Resource,
+			Method:   deleteMethod,
+			// appending to the resource name ensures it is valid
```
Comment (ericwenn) [inline internal/aiptest/deletion/exists.go:34]: this comment is needed here

**Review by ericwenn — APPROVED** [review]
Lovely!

How about a test for checking that `delete_time` / `deleted` is set if the method is for soft delete?

File: internal/aiptest/deletion/exists.go (hunk 1 above)
Comment (radhus) [reply internal/aiptest/deletion/exists.go]: Oops, copy-paste error. Removed now.

**Comment by radhus** [comment]
> How about a test for checking that `delete_time` / `deleted` is set if the method is for soft delete?

Good idea! I started working on a local branch with soft-delete tests, including for Get and List, but I didn't spend too much time on it. The only requirement in [AIP-164](https://google.aip.dev/164) though is that the Delete-endpoint return type should be the resource and not `google.protobuf.Empty` 🤔 

Maybe I can send a (draft) PR later where we can discuss soft-delete 😄

**Pull request merged**